packages/amesos2/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ SET(${PACKAGE_NAME_UC}_RELEASE_DATE "07/28/2011")
 
 IF (${PACKAGE_NAME}_ENABLE_MUMPS AND NOT Tpetra_INST_INT_INT)
    MESSAGE(WARNING "*****Amesos2 will not provide MUMPS Support with Tpetra_INST_INT_INT OFF.*****")
+  SET(HAVE_${PACKAGE_NAME_UC}_MUMPS OFF)
```

Comment:
NOTE: This special logic that is not supported directly by TriBITS but it seems to be what is desired.